abelta/videos
# Installed MSW.

Installed library MSW.
Mocked api calls are now available.

How to test
------------

* Start application
* See that a list of videos is available now on the home page (these are returned from a mocked API call).
* Open console
* Notice message "[MSW] Mocking enabled."
